onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -445,50 +450,45 @@ QNNExecutionProvider::GetSupportedNodes(const GraphViewer& graph_viewer,
       continue;
     }
 
-    if (node_unit_supported_result.count(node_unit) != 0) {
+    if (handled_node_units.count(node_unit) != 0) {
       continue;  // Already handled this node unit
     }
 
-    // Try to convert certain standalone DQ -> Q sequences into QNN Convert op
-    auto convert_result = TryHandleConvertSequence(qnn_model_wrapper,
-                                                   *node_unit,
-                                                   node_unit_map,
-                                                   logger,
-                                                   true /*do_op_validation*/);
-    if (!convert_result.status.IsOK()) {
-      LOGS(logger, WARNING) << "Failed to convert DQ -> Q sequence to QNN Convert. "
-                            << "Type: " << node_unit->OpType() << ", Node name: " << node_unit->Name() << ", "
-                            << "Message: " << convert_result.status.ErrorMessage();
-    }
-
-    bool supported = false;
+    // Try to see if this node unit can be fused.
+    std::vector<const NodeUnit*> fused_nodes;
+    Status fusion_status = TryFusions(fused_nodes, qnn_model_wrapper, *node_unit, node_unit_map,
+                                      handled_node_units, logger, false);
```

Comment:
>false [](http://example.com/codeflow?start=66&length=5)

why do we still need this flag for op_validation if it's always false?

Reply:
This should be true for the validation pass (dont want to actually create qnn node). Thanks for catching this. I'll make the change.